src/main/webapp/js/statusMessage.js

```diff
@@ -1,7 +1,13 @@
 $(document).ready(function() {
     var statusMessage = document.getElementById('statusMessage');
-    var navbarHeight = document.getElementsByClassName('navbar')[0].offsetHeight;
+    var navbarHeight = 0;
     var extraPadding = 15;
+    
+    var navbar = document.getElementsByClassName('navbar')[0];
+    
+    if (navbar != null) {
+        navbarHeight = navbar.offsetHeight;
```

Comment:
@taniach look here, offsetHeight and height are different, offsetHeight takes into account border and padding whereas .height() [for jQuery] or .clientHeight [vanilla JavaScript] returns the exact height of the element sans border and padding so we might want to make a distinction between the two of them for clarity.

But again, considering the urgency of this fix this isn't that important.